test/runtests.jl

```diff
@@ -31,8 +31,15 @@ end
     @test !Coverage.iscovfile("/otherdir/test.jl.cov", "/somedir/test.jl")
 end
 
-@testset "isfuncexpr" begin
+@test Coverage.isfuncexpr(:(f() = x))
```

Comment:
Should be indented to match the others inside of the test set (assuming you did not intend to remove that)